app/components/common/collectionSnackBar/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Location } from 'history';
+import Snackbar from '@material-ui/core/Snackbar';
+import { AppState } from '../../../reducers';
+import { closeCollectionSnackBar } from '../../../reducers/collectionSnackBar';
+import { closeDialog } from '../../dialog/actions';
+import Icon from '../../../icons';
+import ActionTicketManager from '../../../helpers/actionTicketManager';
+import { getCurrentPageType } from '../../locationListener';
+import { UserDevice } from '../../layouts/reducer';
+import Button from '../button';
+const useStyles = require('isomorphic-style-loader/useStyles');
+const s = require('./collectionSnackBar.scss');
+
+type Props = ReturnType<typeof mapStateToProps> & {
+  location: Location;
+};
+
+const CollectionSnackBar: React.FC<Props> = props => {
+  useStyles(s);
+  const dispatch = useDispatch();
+  const { collectionSnackBarState, location, layout } = props;
+  const { collectionId, collectionName, isOpen } = collectionSnackBarState;
+  const isLongName = collectionName.length >= 30 && layout.userDevice === UserDevice.MOBILE;
+
+  useEffect(
+    () => {
+      dispatch(closeCollectionSnackBar());
+    },
+    [dispatch, location]
+  );
+
+  useEffect(
+    () => {
+      if (isOpen) {
+        ActionTicketManager.trackTicket({
+          pageType: getCurrentPageType(),
+          actionType: 'view',
+          actionArea: 'collectionSnackbar',
+          actionTag: 'viewCollectionSnackBar',
+          actionLabel: String(collectionId),
+        });
+      }
+    },
+    [isOpen, collectionId]
+  );
+
+  return (
+    <Snackbar
```

Comment:
is there any special reason to choose SnackBar rather than Notie?
we already have `ReduxNotifier` middleware.